File: Unit1.pas
unit Unit1;

interface

uses
  Winapi.Windows, Winapi.Messages, System.SysUtils, System.Variants, System.Classes, Vcl.Graphics,
  Vcl.Controls, Vcl.Forms, Vcl.Dialogs, Vcl.StdCtrls, Vcl.ExtCtrls;

type
  TForm1 = class(TForm)
    Panel1: TPanel;
    ImageSS: TImage;
    Label1: TLabel;
    editFilePath: TEdit;
    ButtonTakeSS: TButton;
    Bevel1: TBevel;
    Label2: TLabel;
    EditHttpRequest: TEdit;
    ButtonHttpRequest: TButton;
    MemoHttpResponse: TMemo;
    Bevel2: TBevel;
    Label3: TLabel;
    EditSMTPServer: TEdit;
    Label4: TLabel;
    EditPort: TEdit;
    Label5: TLabel;
    EditUsername: TEdit;
    EditPassword: TEdit;
    Label6: TLabel;
    Label7: TLabel;
    EditEmailTo: TEdit;
    EditSubject: TEdit;
    Label8: TLabel;
    MemoBodyMsg: TMemo;
    Label9: TLabel;
    ButtonSendMail: TButton;
    LabelEmailStatus: TLabel;
  private
    { Private declarations }
  public
    { Public declarations }
  end;

var
  Form1: TForm1;

implementation

{$R *.dfm}

end.

File: Unit1.py
import os
from delphivcl import *

class Form1(Form):

    def __init__(self, owner):
        self.Panel1 = None
        self.ImageSS = None
        self.Label1 = None
        self.editFilePath = None
        self.ButtonTakeSS = None
        self.Bevel1 = None
        self.Label2 = None
        self.EditHttpRequest = None
        self.ButtonHttpRequest = None
        self.MemoHttpResponse = None
        self.Bevel2 = None
        self.Label3 = None
        self.EditSMTPServer = None
        self.Label4 = None
        self.EditPort = None
        self.Label5 = None
        self.EditUsername = None
        self.EditPassword = None
        self.Label6 = None
        self.Label7 = None
        self.EditEmailTo = None
        self.EditSubject = None
        self.Label8 = None
        self.MemoBodyMsg = None
        self.Label9 = None
        self.ButtonSendMail = None
        self.LabelEmailStatus = None
        self.LoadProps(os.path.join(os.path.dirname(os.path.abspath(__file__)), "Unit1.pydfm"))

def main():
    Application.Initialize()
    Application.Title = 'Form1'
    MainForm = Form1(Application)
    MainForm.Show()
    FreeConsole()
    Application.Run()

if __name__ == '__main__':
    main()
